src/sign/generic.rs

```diff
@@ -0,0 +1,512 @@
+use core::{fmt, mem, str};
```

Comment:
General tip: please follow the style guidelines at https://github.com/NLnetLabs/domain/blob/contributing-doc/CONTRIBUTING.md.

Reply:
Right, I should have seen that.  Will fix.

Reply:
It's not that easy to find as it's in a branch so I'm not surprised you hadn't seen it, we should approve and merge it.